### PR TITLE
Add WorkflowRegistry to config

### PR DESCRIPTION
**Pull request by cedric-cordenier**
Add config options to configure the WorkflowRegistry.

### CODE REVIEW

**Review by MStreet3 — APPROVED** [review]

**Review by ettec — APPROVED** [review]

**Review by george-dorin — APPROVED** [review]

File: core/services/chainlink/testdata/config-full.toml [hunk 1]
```diff
@@ -279,6 +279,11 @@ Address = ''
 NetworkID = 'evm'
 ChainID = '1'
 
+[Capabilities.WorkflowRegistry]
+Address = ''
```
Comment (chudilka1) [inline core/services/chainlink/testdata/config-full.toml:283]: @cedric-cordenier, does config-`full` imply empty values?

File: docs/CONFIG.md [hunk 2]
```diff
@@ -1213,6 +1213,33 @@ ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
...
+```toml
+Address = '0x0' # Example
+```
+Address is the address for the workflow registry contract.
+
+### NetworkID
+```toml
+NetworkID = 'evm' # Default
+```
+NetworkID identifies the target network where the remote registry is located.
```
Comment (chudilka1) [inline docs/CONFIG.md:1235]: Could you list the other examples of the NetworkID values here as well?

File: core/config/toml/types.go [hunk 3]
```diff
@@ -1452,6 +1452,26 @@ func (r *ExternalRegistry) setFrom(f *ExternalRegistry) {
 	}
 }
 
+type WorkflowRegistry struct {
+	Address   *string
+	NetworkID *string
+	ChainID   *string
+}
+
+func (r *WorkflowRegistry) setFrom(f *WorkflowRegistry) {
```
Comment (chudilka1) [inline core/config/toml/types.go:1461]: Would an unset or empty field invalidate the node's config on boot if the `WorkflowRegistry` is defined in the config but is missing a field or value?
Please reflect it (the invalid configuration) in your `test data/scripts` by showing what would be an invalid way of setting the config.

File: core/services/chainlink/testdata/config-full.toml (hunk 1 above)
Comment (cedric-cordenier) [reply core/services/chainlink/testdata/config-full.toml:283]: It just means that all values have been set explicitly

File: core/config/toml/types.go (hunk 3 above)
Comment (cedric-cordenier) [reply core/config/toml/types.go:1461]: No, these are optional values so they can be unset.

**Pull request merged**